go.mod

```diff
@@ -30,9 +30,11 @@ require (
 	github.com/shurcooL/vfsgen v0.0.0-20180825020608-02ddb050ef6b
 	github.com/stretchr/testify v1.3.0
 	github.com/xlab/treeprint v0.0.0-20180616005107-d6fb6747feb6
-	golang.org/x/net v0.0.0-20181201002055-351d144fa1fc
-	golang.org/x/text v0.3.1-0.20180805044716-cb6730876b98 // indirect
-	golang.org/x/tools v0.0.0-20190118193359-16909d206f00
+	golang.org/x/crypto v0.0.0-20190701094942-4def268fd1a4 // indirect
+	golang.org/x/net v0.0.0-20190724013045-ca1201d0de80
+	golang.org/x/sys v0.0.0-20190801041406-cbf593c0f2f3 // indirect
```

Comment:
From `github.com/prometheus/client_golang/prometheus` which is already enabled for Go modules.